### DISCUSSION TITLE
Code transpilation to Julia (WIP)

### DISCUSSION THREAD

**Discussion by phineas-pta** [krahets/hello-algo#1050] [Show and tell]
hello 👋

any Julia enthusiasts here ?

im currently transpiling the code to Julia: https://github.com/phineas-pta/hello-algo.jl

any help is welcome 🤗

i dont open PR yet because it's still quick dirty implemention, i haven't checked output correctness (also i use unconventional coding style)

**Comment by hpstory** [comment]
Hi @phineas-pta , feel free to reference #15 and [comment](https://github.com/krahets/hello-algo/issues/15#issuecomment-1792081081), cc @krahets .

**Reply by samwelzakayo** [reply]
that's so cool!!!!
